### PR TITLE
[Issue-124] Remove use of PIC from samples

### LINKED ISSUE #124: Don't use PiC functions in samples
We don't want to push users to write code a certain way when using our samples, we should rewrite to look like idiomatic C.

We can keep the same patterns of `CleanUp:` but CHK_STATUS and other things like that should be avoided.

### PR DESCRIPTION
**Pull request by disa6302**
*Issue #, if available: #124 *

*Description of changes:*

- Replaced use of PIC Functions such as CHK_STATUS, CHK and CHK_ERROR with standard C calls from sample
- Removed used of other PiC functions wherever applicable and replaced with standard C functions.
- Removed use of DLOG functions. 

Resolves #124 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by alexw91** [comment]
# [Codecov](https://codecov.io/gh/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/pull/227?src=pr&el=h1) Report
> Merging [#227](https://codecov.io/gh/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/pull/227?src=pr&el=desc) into [master](https://codecov.io/gh/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/commit/07944394f449327f1055449541a6847c82424af8&el=desc) will **increase** coverage by `0.14%`.
> The diff coverage is `n/a`.

[Impacted file tree graph](https://codecov.io/gh/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/pull/227?src=pr&el=tree)

```diff
@@            Coverage Diff             @@
##           master     #227      +/-   ##
==========================================
+ Coverage   86.83%   86.98%   +0.14%     
==========================================
  Files          33       33              
  Lines        7807     7820      +13     
==========================================
+ Hits         6779     6802      +23     
+ Misses       1028     1018      -10     
```


| [Impacted Files](https://codecov.io/gh/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/pull/227?src=pr&el=tree) | Coverage Δ | |
|---|---|---|
| [src/source/Ice/IceAgentStateMachine.c](https://codecov.io/gh/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/pull/227/diff?src=pr&el=tree#diff-c3JjL3NvdXJjZS9JY2UvSWNlQWdlbnRTdGF0ZU1hY2hpbmUuYw==) | `83.33% <0.00%> (-0.59%)` | :arrow_down: |
| [src/source/PeerConnection/JitterBuffer.c](https://codecov.io/gh/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/pull/227/diff?src=pr&el=tree#diff-c3JjL3NvdXJjZS9QZWVyQ29ubmVjdGlvbi9KaXR0ZXJCdWZmZXIuYw==) | `95.48% <0.00%> (+0.05%)` | :arrow_up: |
| [src/source/Ice/IceAgent.c](https://codecov.io/gh/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/pull/227/diff?src=pr&el=tree#diff-c3JjL3NvdXJjZS9JY2UvSWNlQWdlbnQuYw==) | `93.99% <0.00%> (+0.10%)` | :arrow_up: |
| [src/source/PeerConnection/PeerConnection.c](https://codecov.io/gh/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/pull/227/diff?src=pr&el=tree#diff-c3JjL3NvdXJjZS9QZWVyQ29ubmVjdGlvbi9QZWVyQ29ubmVjdGlvbi5j) | `93.94% <0.00%> (+0.66%)` | :arrow_up: |
| [src/source/PeerConnection/SessionDescription.c](https://codecov.io/gh/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/pull/227/diff?src=pr&el=tree#diff-c3JjL3NvdXJjZS9QZWVyQ29ubmVjdGlvbi9TZXNzaW9uRGVzY3JpcHRpb24uYw==) | `83.82% <0.00%> (+1.98%)` | :arrow_up: |

------

[Continue to review full report at Codecov](https://codecov.io/gh/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/pull/227?src=pr&el=continue).
> **Legend** - [Click here to learn more](https://docs.codecov.io/docs/codecov-delta)
> `Δ = absolute <relative> (impact)`, `ø = not affected`, `? = missing data`
> Powered by [Codecov](https://codecov.io/gh/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/pull/227?src=pr&el=footer). Last update [c8d8426...753cc5b](https://codecov.io/gh/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/pull/227?src=pr&el=lastupdated). Read the [comment docs](https://docs.codecov.io/docs/pull-request-comments).

**Comment by alexw91** [comment]
# [Codecov](https://codecov.io/gh/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/pull/227?src=pr&el=h1) Report
> Merging [#227](https://codecov.io/gh/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/pull/227?src=pr&el=desc) into [master](https://codecov.io/gh/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/commit/07944394f449327f1055449541a6847c82424af8&el=desc) will **increase** coverage by `0.14%`.
> The diff coverage is `n/a`.

[Impacted file tree graph](https://codecov.io/gh/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/pull/227?src=pr&el=tree)

```diff
@@            Coverage Diff             @@
##           master     #227      +/-   ##
==========================================
+ Coverage   86.83%   86.98%   +0.14%     
==========================================
  Files          33       33              
  Lines        7807     7820      +13     
==========================================
+ Hits         6779     6802      +23     
+ Misses       1028     1018      -10     
```


| [Impacted Files](https://codecov.io/gh/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/pull/227?src=pr&el=tree) | Coverage Δ | |
|---|---|---|
| [src/source/Ice/IceAgentStateMachine.c](https://codecov.io/gh/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/pull/227/diff?src=pr&el=tree#diff-c3JjL3NvdXJjZS9JY2UvSWNlQWdlbnRTdGF0ZU1hY2hpbmUuYw==) | `83.33% <0.00%> (-0.59%)` | :arrow_down: |
| [src/source/PeerConnection/JitterBuffer.c](https://codecov.io/gh/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/pull/227/diff?src=pr&el=tree#diff-c3JjL3NvdXJjZS9QZWVyQ29ubmVjdGlvbi9KaXR0ZXJCdWZmZXIuYw==) | `95.48% <0.00%> (+0.05%)` | :arrow_up: |
| [src/source/Ice/IceAgent.c](https://codecov.io/gh/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/pull/227/diff?src=pr&el=tree#diff-c3JjL3NvdXJjZS9JY2UvSWNlQWdlbnQuYw==) | `93.99% <0.00%> (+0.10%)` | :arrow_up: |
| [src/source/PeerConnection/PeerConnection.c](https://codecov.io/gh/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/pull/227/diff?src=pr&el=tree#diff-c3JjL3NvdXJjZS9QZWVyQ29ubmVjdGlvbi9QZWVyQ29ubmVjdGlvbi5j) | `93.94% <0.00%> (+0.66%)` | :arrow_up: |
| [src/source/PeerConnection/SessionDescription.c](https://codecov.io/gh/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/pull/227/diff?src=pr&el=tree#diff-c3JjL3NvdXJjZS9QZWVyQ29ubmVjdGlvbi9TZXNzaW9uRGVzY3JpcHRpb24uYw==) | `83.82% <0.00%> (+1.98%)` | :arrow_up: |

------

[Continue to review full report at Codecov](https://codecov.io/gh/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/pull/227?src=pr&el=continue).
> **Legend** - [Click here to learn more](https://docs.codecov.io/docs/codecov-delta)
> `Δ = absolute <relative> (impact)`, `ø = not affected`, `? = missing data`
> Powered by [Codecov](https://codecov.io/gh/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/pull/227?src=pr&el=footer). Last update [c8d8426...6419b06](https://codecov.io/gh/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/pull/227?src=pr&el=lastupdated). Read the [comment docs](https://docs.codecov.io/docs/pull-request-comments).

**Comment by alexw91** [comment]
# [Codecov](https://codecov.io/gh/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/pull/227?src=pr&el=h1) Report
> Merging [#227](https://codecov.io/gh/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/pull/227?src=pr&el=desc) into [master](https://codecov.io/gh/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/commit/07944394f449327f1055449541a6847c82424af8&el=desc) will **increase** coverage by `0.13%`.
> The diff coverage is `n/a`.

[Impacted file tree graph](https://codecov.io/gh/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/pull/227?src=pr&el=tree)

```diff
@@            Coverage Diff             @@
##           master     #227      +/-   ##
==========================================
+ Coverage   86.83%   86.96%   +0.13%     
==========================================
  Files          33       33              
  Lines        7807     7820      +13     
==========================================
+ Hits         6779     6801      +22     
+ Misses       1028     1019       -9     
```


| [Impacted Files](https://codecov.io/gh/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/pull/227?src=pr&el=tree) | Coverage Δ | |
|---|---|---|
| [src/source/Ice/IceAgentStateMachine.c](https://codecov.io/gh/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/pull/227/diff?src=pr&el=tree#diff-c3JjL3NvdXJjZS9JY2UvSWNlQWdlbnRTdGF0ZU1hY2hpbmUuYw==) | `83.33% <0.00%> (-0.59%)` | :arrow_down: |
| [src/source/PeerConnection/JitterBuffer.c](https://codecov.io/gh/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/pull/227/diff?src=pr&el=tree#diff-c3JjL3NvdXJjZS9QZWVyQ29ubmVjdGlvbi9KaXR0ZXJCdWZmZXIuYw==) | `95.48% <0.00%> (+0.05%)` | :arrow_up: |
| [src/source/PeerConnection/PeerConnection.c](https://codecov.io/gh/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/pull/227/diff?src=pr&el=tree#diff-c3JjL3NvdXJjZS9QZWVyQ29ubmVjdGlvbi9QZWVyQ29ubmVjdGlvbi5j) | `93.94% <0.00%> (+0.66%)` | :arrow_up: |
| [src/source/PeerConnection/SessionDescription.c](https://codecov.io/gh/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/pull/227/diff?src=pr&el=tree#diff-c3JjL3NvdXJjZS9QZWVyQ29ubmVjdGlvbi9TZXNzaW9uRGVzY3JpcHRpb24uYw==) | `83.82% <0.00%> (+1.98%)` | :arrow_up: |

------

[Continue to review full report at Codecov](https://codecov.io/gh/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/pull/227?src=pr&el=continue).
> **Legend** - [Click here to learn more](https://docs.codecov.io/docs/codecov-delta)
> `Δ = absolute <relative> (impact)`, `ø = not affected`, `? = missing data`
> Powered by [Codecov](https://codecov.io/gh/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/pull/227?src=pr&el=footer). Last update [c8d8426...dd2db10](https://codecov.io/gh/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/pull/227?src=pr&el=lastupdated). Read the [comment docs](https://docs.codecov.io/docs/pull-request-comments).

**Comment by alexw91** [comment]
# [Codecov](https://codecov.io/gh/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/pull/227?src=pr&el=h1) Report
> Merging [#227](https://codecov.io/gh/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/pull/227?src=pr&el=desc) into [master](https://codecov.io/gh/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/commit/07944394f449327f1055449541a6847c82424af8&el=desc) will **increase** coverage by `0.14%`.
> The diff coverage is `n/a`.

[Impacted file tree graph](https://codecov.io/gh/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/pull/227?src=pr&el=tree)

```diff
@@            Coverage Diff             @@
##           master     #227      +/-   ##
==========================================
+ Coverage   86.83%   86.98%   +0.14%     
==========================================
  Files          33       33              
  Lines        7807     7820      +13     
==========================================
+ Hits         6779     6802      +23     
+ Misses       1028     1018      -10     
```


| [Impacted Files](https://codecov.io/gh/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/pull/227?src=pr&el=tree) | Coverage Δ | |
|---|---|---|
| [src/source/Ice/IceAgentStateMachine.c](https://codecov.io/gh/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/pull/227/diff?src=pr&el=tree#diff-c3JjL3NvdXJjZS9JY2UvSWNlQWdlbnRTdGF0ZU1hY2hpbmUuYw==) | `83.33% <0.00%> (-0.59%)` | :arrow_down: |
| [src/source/PeerConnection/JitterBuffer.c](https://codecov.io/gh/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/pull/227/diff?src=pr&el=tree#diff-c3JjL3NvdXJjZS9QZWVyQ29ubmVjdGlvbi9KaXR0ZXJCdWZmZXIuYw==) | `95.48% <0.00%> (+0.05%)` | :arrow_up: |
| [src/source/Ice/IceAgent.c](https://codecov.io/gh/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/pull/227/diff?src=pr&el=tree#diff-c3JjL3NvdXJjZS9JY2UvSWNlQWdlbnQuYw==) | `93.99% <0.00%> (+0.10%)` | :arrow_up: |
| [src/source/PeerConnection/PeerConnection.c](https://codecov.io/gh/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/pull/227/diff?src=pr&el=tree#diff-c3JjL3NvdXJjZS9QZWVyQ29ubmVjdGlvbi9QZWVyQ29ubmVjdGlvbi5j) | `93.94% <0.00%> (+0.66%)` | :arrow_up: |
| [src/source/PeerConnection/SessionDescription.c](https://codecov.io/gh/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/pull/227/diff?src=pr&el=tree#diff-c3JjL3NvdXJjZS9QZWVyQ29ubmVjdGlvbi9TZXNzaW9uRGVzY3JpcHRpb24uYw==) | `83.82% <0.00%> (+1.98%)` | :arrow_up: |

------

[Continue to review full report at Codecov](https://codecov.io/gh/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/pull/227?src=pr&el=continue).
> **Legend** - [Click here to learn more](https://docs.codecov.io/docs/codecov-delta)
> `Δ = absolute <relative> (impact)`, `ø = not affected`, `? = missing data`
> Powered by [Codecov](https://codecov.io/gh/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/pull/227?src=pr&el=footer). Last update [c8d8426...47cde2f](https://codecov.io/gh/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/pull/227?src=pr&el=lastupdated). Read the [comment docs](https://docs.codecov.io/docs/pull-request-comments).

File: samples/kvsWebRTCClientMaster.c [hunk 1]
```diff
@@ -178,59 +243,81 @@ PVOID sendAudioPackets(PVOID args)
...
             ATOMIC_INCREMENT(&pSampleConfiguration->streamingSessionListReadingThreadCount);
             for (i = 0; i < pSampleConfiguration->streamingSessionCount; ++i) {
                 status = writeFrame(pSampleConfiguration->sampleStreamingSessionList[i]->pAudioRtcRtpTransceiver, &frame);
-                if (STATUS_FAILED(status)) {
-                    DLOGD("writeFrame failed with 0x%08x", status);
+                if (status != STATUS_SUCCESS) {
+                    printf("writeFrame failed with 0x%08x", status);
                 }
             }
             ATOMIC_DECREMENT(&pSampleConfiguration->streamingSessionListReadingThreadCount);
```
Comment (sharadrj) [inline samples/kvsWebRTCClientMaster.c]: Is this also a PIC macro?

Comment (disa6302) [reply samples/kvsWebRTCClientMaster.c]: Yes. There are a few atomic operations throughout the file which are platform dependent and that is why I did not replace them yet. We need to think about how to deal with them.

File: samples/kvsWebRTCClientMaster.c [hunk 2]
```diff
@@ -178,59 +243,81 @@ PVOID sendAudioPackets(PVOID args)
...
         }
-
-        THREAD_SLEEP(SAMPLE_AUDIO_FRAME_DURATION);
+        usleep(SAMPLE_AUDIO_FRAME_DURATION / HUNDREDS_OF_NANOS_IN_A_MICROSECOND);
```
Comment (MushMal) [inline samples/kvsWebRTCClientMaster.c]: Non-windows portalbe.

File: samples/kvsWebRTCClientViewer.c [hunk 3]
```diff
@@ -32,45 +42,85 @@ INT32 main(INT32 argc, CHAR *argv[])
...
-    CHK_STATUS(createSignalingClientSync(&clientInfo,
-            &pSampleConfiguration->channelInfo,
-            &signalingClientCallbacks,
-            pSampleConfiguration->pCredentialProvider,
-            &pSampleConfiguration->signalingClientHandle));
     printf("[KVS Viewer] Signaling client created successfully\n");
 
     // Initialize streaming session
-    MUTEX_LOCK(pSampleConfiguration->sampleConfigurationObjLock);
+    pthread_mutex_lock((pthread_mutex_t*) pSampleConfiguration->sampleConfigurationObjLock);
```
Comment (MushMal) [inline samples/kvsWebRTCClientViewer.c]: This is not portable

File: samples/kvsWebRTCClientViewer.c [hunk 4]
```diff
@@ -32,45 +42,85 @@ INT32 main(INT32 argc, CHAR *argv[])
...
+    if(retStatus != STATUS_SUCCESS) {
+        printf("[KVS Viewer] createSampleStreamingSession(): operation returned status code: 0x%08x \n", retStatus);
+        goto CleanUp;
+    }
+
     printf("[KVS Viewer] Creating streaming session...completed\n");
     pSampleConfiguration->sampleStreamingSessionList[pSampleConfiguration->streamingSessionCount++] = pSampleStreamingSession;
 
-    MUTEX_UNLOCK(pSampleConfiguration->sampleConfigurationObjLock);
+    pthread_mutex_unlock((pthread_mutex_t*) pSampleConfiguration->sampleConfigurationObjLock);
```
Comment (MushMal) [inline samples/kvsWebRTCClientViewer.c]: ditto

File: samples/kvsWebRTCClientViewer.c [hunk 5]
```diff
@@ -79,22 +129,37 @@ INT32 main(INT32 argc, CHAR *argv[])
...
     // Block forever
     THREAD_SLEEP(MAX_UINT64);
 
 CleanUp:
     printf("[KVS Viewer] Cleaning up....\n");
-    CHK_LOG_ERR_NV(retStatus);
 
     if (locked) {
-        MUTEX_UNLOCK(pSampleConfiguration->sampleConfigurationObjLock);
+        pthread_mutex_unlock((pthread_mutex_t*) pSampleConfiguration->sampleConfigurationObjLock);
```
Comment (MushMal) [inline samples/kvsWebRTCClientViewer.c]: There is an implementation of ptrheads for most platforms but this is similar to other PIC concepts - do we need to remove the portable public PIC api in favor of non-portable implementation?

**Comment by alexw91** [comment]
# [Codecov](https://codecov.io/gh/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/pull/227?src=pr&el=h1) Report
> Merging [#227](https://codecov.io/gh/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/pull/227?src=pr&el=desc) into [master](https://codecov.io/gh/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/commit/07944394f449327f1055449541a6847c82424af8&el=desc) will **increase** coverage by `0.19%`.
> The diff coverage is `n/a`.

[Impacted file tree graph](https://codecov.io/gh/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/pull/227?src=pr&el=tree)

```diff
@@            Coverage Diff             @@
##           master     #227      +/-   ##
==========================================
+ Coverage   86.83%   87.02%   +0.19%     
==========================================
  Files          33       33              
  Lines        7807     7837      +30     
==========================================
+ Hits         6779     6820      +41     
+ Misses       1028     1017      -11     
```


| [Impacted Files](https://codecov.io/gh/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/pull/227?src=pr&el=tree) | Coverage Δ | |
|---|---|---|
| [src/source/Ice/IceAgentStateMachine.c](https://codecov.io/gh/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/pull/227/diff?src=pr&el=tree#diff-c3JjL3NvdXJjZS9JY2UvSWNlQWdlbnRTdGF0ZU1hY2hpbmUuYw==) | `83.33% <0.00%> (-0.59%)` | :arrow_down: |
| [src/source/Ice/IceAgent.c](https://codecov.io/gh/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/pull/227/diff?src=pr&el=tree#diff-c3JjL3NvdXJjZS9JY2UvSWNlQWdlbnQuYw==) | `93.70% <0.00%> (-0.19%)` | :arrow_down: |
| [src/source/PeerConnection/Rtcp.c](https://codecov.io/gh/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/pull/227/diff?src=pr&el=tree#diff-c3JjL3NvdXJjZS9QZWVyQ29ubmVjdGlvbi9SdGNwLmM=) | `30.55% <0.00%> (ø)` | |
| [src/source/PeerConnection/JitterBuffer.c](https://codecov.io/gh/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/pull/227/diff?src=pr&el=tree#diff-c3JjL3NvdXJjZS9QZWVyQ29ubmVjdGlvbi9KaXR0ZXJCdWZmZXIuYw==) | `95.48% <0.00%> (+0.05%)` | :arrow_up: |
| [src/source/Dtls/Dtls.c](https://codecov.io/gh/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/pull/227/diff?src=pr&el=tree#diff-c3JjL3NvdXJjZS9EdGxzL0R0bHMuYw==) | `93.72% <0.00%> (+0.17%)` | :arrow_up: |
| [src/source/Signaling/LwsApiCalls.c](https://codecov.io/gh/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/pull/227/diff?src=pr&el=tree#diff-c3JjL3NvdXJjZS9TaWduYWxpbmcvTHdzQXBpQ2FsbHMuYw==) | `84.66% <0.00%> (+0.22%)` | :arrow_up: |
| [src/source/Signaling/StateMachine.c](https://codecov.io/gh/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/pull/227/diff?src=pr&el=tree#diff-c3JjL3NvdXJjZS9TaWduYWxpbmcvU3RhdGVNYWNoaW5lLmM=) | `76.61% <0.00%> (+0.32%)` | :arrow_up: |
| [src/source/PeerConnection/PeerConnection.c](https://codecov.io/gh/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/pull/227/diff?src=pr&el=tree#diff-c3JjL3NvdXJjZS9QZWVyQ29ubmVjdGlvbi9QZWVyQ29ubmVjdGlvbi5j) | `93.94% <0.00%> (+0.66%)` | :arrow_up: |
| [src/source/Signaling/ChannelInfo.c](https://codecov.io/gh/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/pull/227/diff?src=pr&el=tree#diff-c3JjL3NvdXJjZS9TaWduYWxpbmcvQ2hhbm5lbEluZm8uYw==) | `88.48% <0.00%> (+1.43%)` | :arrow_up: |
| [src/source/PeerConnection/SessionDescription.c](https://codecov.io/gh/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/pull/227/diff?src=pr&el=tree#diff-c3JjL3NvdXJjZS9QZWVyQ29ubmVjdGlvbi9TZXNzaW9uRGVzY3JpcHRpb24uYw==) | `83.82% <0.00%> (+1.98%)` | :arrow_up: |

------

[Continue to review full report at Codecov](https://codecov.io/gh/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/pull/227?src=pr&el=continue).
> **Legend** - [Click here to learn more](https://docs.codecov.io/docs/codecov-delta)
> `Δ = absolute <relative> (impact)`, `ø = not affected`, `? = missing data`
> Powered by [Codecov](https://codecov.io/gh/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/pull/227?src=pr&el=footer). Last update [c8d8426...3528f17](https://codecov.io/gh/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/pull/227?src=pr&el=lastupdated). Read the [comment docs](https://docs.codecov.io/docs/pull-request-comments).

**Pull request merged**